README.md

```diff
@@ -465,7 +466,7 @@ Soroban.RPC.get_events(events_payload)
          "AAAADQAAACVVU0RDOl3dfLGIo7lPPO+E0KPPSVxWCQ1qOen8umo/g+Jx8baEAAAA"
        ],
        type: "contract",
-       value: %{xdr: "AAAACgAAAAAF9eEAAAAAAAAAAAA="}
+       value: "AAAACgAAAAAF9eEAAAAAAAAAAAA="
```

Comment:
This change needs to be done in `rpc_test.exs`, `get_events_text.exs`, `get_event_response_test.exs` files. Please review if there are more modules to update.